layers/meta-opentrons/recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -120,6 +120,12 @@ python do_create_opentrons_manifest() {
     opentrons_json_output = "%s/VERSION.json" % d.getVar('DEPLOY_DIR_IMAGE')
     with open(opentrons_json_output, 'w') as fh:
         json.dump(opentrons_manifest, fh, indent=4)
+
+    # write the system version file
+    ot_system_version = "%s/OT_SYSTEM_VERSION" % d.getVar('DEPLOY_DIR_IMAGE')
```

Comment:
The thing is, this fully duplicates an element of `/etc/VERSION.json`. I don't really love adding the extra file.